cpp/daal/src/algorithms/kmeans/kmeans_lloyd_batch_impl.i

```diff
@@ -65,33 +65,6 @@ Status KMeansBatchKernel<method, algorithmFPType, cpu>::compute(const NumericTab
     TArray<algorithmFPType, cpu> clusterS1(nClusters * p);
     DAAL_CHECK(clusterS0.get() && clusterS1.get(), services::ErrorMemoryAllocationFailed);
 
-    /* Categorial variables check and support: begin */
```

Comment:
is this handled somewhere else?

Reply:
It doesnt affect oneDAL code, because we dont use CATEGORIAL parameter in oneDAL tables